File: week-04/day-03/anagram.py
# Anagram

#     Write a function, that takes two strings and returns a boolean value based on if the two strings are Anagramms or not.
#     Create a test for that.

def anagram (word, anag):
    word = word.replace(" ", "") 
    anag = anag.replace(" ", "")    
    if sorted(word.lower()) == sorted(anag.lower()):
        return True
    else:
        return False

File: week-09/day-03/anagram.js
// Anagram
//     Write a function, that takes two strings and 
//     returns a boolean value based on if the two strings are Anagramms or not.
//     Create a test for that.
'use strict';

function anagram(s1, s2){
  return s1.replace(/ /g,'').toLowerCase().split('').sort().join('') === s2.replace(/ /g,'').toLowerCase().split('').sort().join('');
}

module.exports = anagram;
// anagram('William Shakespeare', 'I am a weakish speller')
